inode/file.go

```diff
@@ -617,6 +657,7 @@ func (vS *volumeStruct) Write(fileInodeNumber InodeNumber, offset uint64, buf []
 	fileInode.AttrChangeTime = updateTime
 	fileInode.ModificationTime = updateTime
 	fileInode.NumWrites++
+	fileInode.dirty = true
```

Comment:
I am curious - did you have a specific test which was failing that caused you to find this bug?
Or, simply by reading the code did you notice this was not set correctly?

Reply:
Oh - I see you just moved it down further with the rest of the changes.